.github/workflows/nodejs.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest]
-        node-version: [8.x, 10.x, 12.x]
```

Comment:
ooooo

https://mochajs.org/#installation

>As of v8.0.0, Mocha requires Node.js v10.12.0 or newer.

Reply:
I'm leaning towards the idea that bumping the minimum node to v10 is non-breaking.

Reply:
For reference, Gatsby dropped Node 8 support in a minor release. https://www.gatsbyjs.org/blog/2020-03-20-dropping-support-for-node-8/

Looks like they also had deprecation warnings in a couple releases leading up to the minor bump.

Reply:
Interestinggggg, so it looks like the `mocha` _CLI_ requires Node v10.12.0 and up, but not the API.

After QA'ing a bit, I have not seen this to be an actual breaking change to end users.